### ISSUE TITLE
Object.extname is deprecated.

### ISSUE THREAD

**Issue by oakmac** [oakmac/atom-parinfer#109]
Argument to `path.extname` must be a string
```
Object.extname (/usr/share/atom/resources/app.asar/src/electron-shims.js:20:10)
Zo (/home/oakmac/.atom/packages/parinfer/lib/atom-parinfer.js:515:361)
yp (/home/oakmac/.atom/packages/parinfer/lib/atom-parinfer.js:534:105)
<unknown> (<embedded>:11:428865)
Function.simpleDispatch (<embedded>:11:1201500)
Emitter.emit (<embedded>:11:1202941)
```

**Comment by oakmac** [comment]
Saw this today while using v1.24.0

**Comment by oakmac** [comment]
Fixed with v1.24.1
